src/software/ai/hl/stp/tactic/crease_defender/crease_defender_fsm.cpp

```diff
@@ -30,10 +34,21 @@ std::optional<Point> CreaseDefenderFSM::findBlockThreatPoint(
     return findDefenseAreaIntersection(field, ray, robot_obstacle_inflation_factor);
```

Comment:
You should make `findDefenseAreaIntersection` static so `findBlockThreatPoint` can call it

Reply:
Ohhhh right, that's why I originally wrote it as static